commit af5823b30757722ef4ddedacc7b5cd10d4cf8f98
Author: Peter Huewe <peterhuewe@gmx.de>
Date:   Mon Sep 3 21:51:51 2018 +0200

    tpm: Make SECURITYFS a weak dependency
    
    commit 2f7d8dbb11287cbe9da6380ca14ed5d38c9ed91f upstream.
    
    While having SECURITYFS enabled for the tpm subsystem is beneficial in
    most cases, it is not strictly necessary to have it enabled at all.
    Especially on platforms without any boot firmware integration of the TPM
    (e.g. raspberry pi) it does not add any value for the tpm subsystem,
    as there is no eventlog present.
    
    By turning it from 'select' to 'imply' it still gets selected per
    default, but enables users who want to save some kb of ram by turning
    SECURITYFS off.
    
    Signed-off-by: Peter Huewe <peterhuewe@gmx.de>
    Reviewed-by: Jarkko Sakkinen <jarkko.sakkinen@linux.intel.com>
    Signed-off-by: Jarkko Sakkinen <jarkko.sakkinen@linux.intel.com>

diff --git a/drivers/char/tpm/Kconfig b/drivers/char/tpm/Kconfig
index 18c81cbe4704..536e55d3919f 100644
--- a/drivers/char/tpm/Kconfig
+++ b/drivers/char/tpm/Kconfig
@@ -5,7 +5,7 @@
 menuconfig TCG_TPM
 	tristate "TPM Hardware Support"
 	depends on HAS_IOMEM
-	select SECURITYFS
+	imply SECURITYFS
 	select CRYPTO
 	select CRYPTO_HASH_INFO
 	---help---
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1435_af5823b30757722ef4ddedacc7b5cd10d4cf8f98.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1435_af5823b30757722ef4ddedacc7b5cd10d4cf8f98.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
